www/content/docs/pages.mdx

```diff
@@ -168,15 +134,20 @@ export async function getStaticProps(context) {
     params.addFields("node--article", ["title", "path", "body", "uid"])
   }
 
-  const node = await drupal.getResourceFromContext(type, context, {
+  const node = await drupal.getResourceByPath(path, {
     params: params.getQueryObject(),
   })
 
-  return {
-    props: {
-      node,
-    },
+  // Render different Components based on Node type
+  if (node.type === "node--page") {
+  return <PageComponent />
```

Comment:
We'd have to pass some kind of props here (and also for the article below), if it was the entire node object it would be:

```suggestion
  return <PageComponent node={node} />
```